micro-rdk/src/common/provisioning/server.rs

```diff
@@ -522,7 +522,7 @@ pub trait WifiManager: Network {
         &'a self,
         ssid: &'a str,
         password: &'a str,
-    ) -> Pin<Box<dyn Future<Output = Result<(), WifiManagerError>> + '_>>;
```

Comment:
interesting that this inverse change is needed opposed to the other lifetime changes. I guess because it's the returned result and not the input parameter.